src/3d/qgstessellator.cpp

```diff
@@ -358,6 +359,18 @@ void QgsTessellator::addPolygon( const QgsPolygon &polygon, float extrusionHeigh
       if ( mAddNormals )
         mData << pNormal.x() << pNormal.z() << - pNormal.y();
     }
+
+    if ( mAddBackFaces )
+    {
+      // the same triangle with reversed order of coordinates and inverted normal
+      for ( int i = 2; i >= 0; i-- )
```

Comment:
Marginally more readable would be:

    for ( int i : { 2, 1, 0 } )

Reply:
Umm... I would prefer to stick to my version :-) Maybe I am being just conservative, but the original line is easier for my eyes (and it is consistent with the for cycle adding front faces)

Reply:
Up to you, but to me the list version can be read in 0.5 seconds vs the 3 seconds it takes to read the non-list version :p